src/main/java/org/apache/hadoop/hive/metastore/MetaStoreUtils.java

```diff
@@ -0,0 +1,9 @@
+package org.apache.hadoop.hive.metastore;
+
+/**
+ * Coral depends on a different version of Hive, which has the MetaStoreUtils to a different location. This change
+ * allows Coral to work across Hive versions.
+ * Ref: PRESTO-2013
```

Comment:
nit: we can remove the ticket number :) It'd be good to add any useful details from there to the commit message.

Reply:
Agreed, I was going to ask about this.